warehouse/models/mart/gtfs_quality/fct_monthly_route_id_changes.sql

```diff
@@ -0,0 +1,68 @@
+WITH dim_routes AS (
+    SELECT  * FROM {{ ref('dim_routes') }}
+),
+
+feed_version_history AS (
+    SELECT * FROM {{ ref('int_gtfs_quality__feed_version_history') }}
+),
+
+route_id_comparison AS (
+    SELECT * FROM {{ ids_version_compare_aggregate("route_id","dim_routes") }}
+),
+
+date_range AS (
+    SELECT DISTINCT date_start, date_end, publish_date FROM {{ ref('idx_monthly_reports_site') }}
+),
+
+month_start AS (
+    SELECT
+        base64_url,
+        feed_key,
+        id,
+        publish_date
+    FROM route_id_comparison
```

Comment:
I don't follow why `route_id_comparison` is needed here -- it doesn't look like you are keeping any of the route comparison columns.